### ISSUE TITLE
Tdarr_Plugin_bsh1_Boosh_FFMPEG_QSV_HEVC doesn't handle undefined tags

### ISSUE THREAD

**Issue by vendruscolo** [HaveAGitGat/Tdarr_Plugins#581]
This is what gets logged:

```
2024-01-04T11:17:29.454Z Pre-processing - Tdarr_Plugin_bsh1_Boosh_FFMPEG_QSV_HEVC☒Plugin error! TypeError: Cannot read properties of undefined (reading 'tags')
```

I'm attaching the movie JSON rep it was trying to process (recorded from Plex in a TS container):
[data.txt](https://github.com/HaveAGitGat/Tdarr_Plugins/files/13829421/data.txt)

I can see none of the tracks in `mediaInfo.track` have the `tags` property. Other recordings had it, I don't understand why there's this difference. Anyway, I can make a PR to fix this issue, however I don't understand which line is causing it. Would verbose logging help in this case?

**Comment by Boosh1** [comment]
@vendruscolo Not sure if you were able to solve this on your end but I think my latest pull request will resolve this
https://github.com/HaveAGitGat/Tdarr_Plugins/pull/584

**Comment by vendruscolo** [comment]
@Boosh1 yeah I solved it and verified it works for my Plex recordings. I planned to put up a PR but was busy and still haven't done it. I'll look into yours and eventually leave some feedback. Thanks!

**Comment by HaveAGitGat** [comment]
Closing as PR was merged
